workspaces/geor/automated_test/datastore.xml

```diff
@@ -0,0 +1,23 @@
+<dataStore>
+  <id>DataStoreInfoImpl-2b665931:15a1d6fbd50:-7ff7</id>
+  <name>automated_test</name>
+  <description>Dataset for automated test (please do not remove)</description>
+  <type>Shapefile</type>
+  <enabled>true</enabled>
+  <workspace>
+    <id>WorkspaceInfoImpl-2b665931:15a1d6fbd50:-7ff9</id>
+  </workspace>
+  <connectionParameters>
+    <entry key="memory mapped buffer">false</entry>
+    <entry key="timezone">Etc/UTC</entry>
+    <entry key="fstype">shape</entry>
```

Comment:
I would have prefered a repo typed "directory of shapefiles" rather than "shapefile".
It allows for more layers (for more automated tests to come)